### PR TITLE
Fix M365 Usage Report Graph V1 Table

### PR DESCRIPTION
**Pull request by airicaruss**
Fix M365 Usage Report Graph V1 Table incorrectly add check mark for Germany environment.

### CODE REVIEW

**Review by Danielabom — APPROVED** [review]
No schema changes as per @airicaruss. PR approved.

File: api-reference/v1.0/resources/report.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The following table shows the availability for each API across all cloud deploym
...
 | [Microsoft 365 groups activity](/graph/api/resources/office-365-groups-activity-reports?view=graph-rest-1.0&preserve-view=true) | ✔                              | ➖                                     | ➖                                              | ➖                           |
 | [Microsoft Teams device usage](/graph/api/resources/microsoft-teams-device-usage-reports?view=graph-rest-1.0&preserve-view=true) | ✔                              | ➖                                     | ➖                                              | ➖                           |
-| [Microsoft Teams team usage](/graph/api/resources/microsoft-teams-team-usage-reports?view=graph-rest-1.0&preserve-view=true) | ✔                              | ➖                                     | ➖                                              | ➖                           |  ✔                              |
+| [Microsoft Teams team usage](/graph/api/resources/microsoft-teams-team-usage-reports?view=graph-rest-1.0&preserve-view=true) | ✔                              | ➖                                     | ➖                                              | ➖                           |
```
Comment (Lauragra) [inline api-reference/v1.0/resources/report.md:31]: Before we merge this, can you please verify that we have a Microsoft cloud for Germany? I think we might need to remove the Germany column.

Comment (airicaruss) [reply api-reference/v1.0/resources/report.md:31]: We don't have such. I will remove this column.

Comment (airicaruss) [reply api-reference/v1.0/resources/report.md]: Removed Germany environment from the table

**Review by Lauragra — APPROVED** [review]

**Pull request merged**